src/lokalise/push/client.ts

```diff
@@ -48,25 +48,36 @@ export class LokalisePushClient extends LokaliseClient {
       return;
     }
 
-    const filepath = join(fileDirectory, fileName);
-    const content = await promises.readFile(filepath, { encoding: 'base64' });
+    try {
+      // In Node 12, the is no convention for checking if a file exists before reading it.
+      // Instead, attempt to read the file and catch any errors
+      // https://nodejs.org/docs/latest-v12.x/api/fs.html#fs_fspromises_access_path_mode
+      const filepath = join(fileDirectory, fileName);
```

Comment:
Looks like you just need to move this out of the try block.. and maybe correct small typo in comment (the -> there)